snap/hooks/remove

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+TAG="openthread-border-router.remove"
+
+logger --tag=$TAG "Start"
+
+
+###############################################################################
+logger --tag=$TAG "Remove the firewall config"
+$SNAP/bin/script/otbr-firewall stop
+
+# Custom initrc without staging and build logic
+export PLATFORM=ubuntu
+. $SNAP/bin/_initrc_install
```

Comment:
There is no explicit code in this script for removing or uninstalling any software or components. Therefore, it is unclear why it would be included in a remove hook.

Reply:
The sourced script contains functions that are used by other scripts.